mappings/net/minecraft/client/realms/gui/FetchTask.mapping

```diff
@@ -0,0 +1,22 @@
+CLASS net/minecraft/class_5784 net/minecraft/client/realms/gui/FetchTask
+	COMMENT A task for {@link RealmsDataFetcher} that runs at a fixed interval. It
+	COMMENT optionally supports rate limits and stops when the fetcher is stopped.
+	FIELD field_28452 condition Ljava/util/function/BooleanSupplier;
+	FIELD field_28453 rateLimiter Lnet/minecraft/class_5785;
+	FIELD field_28454 period Ljava/time/Duration;
+	FIELD field_28455 command Ljava/lang/Runnable;
+	METHOD <init> (Ljava/lang/Runnable;Ljava/time/Duration;Ljava/util/function/BooleanSupplier;Lnet/minecraft/class_5785;)V
+		ARG 1 command
+		ARG 2 period
+		ARG 3 condition
+		ARG 4 periodTracker
```

Comment:
`rateLimiter` or `periodTracker`?

Reply:
```suggestion
		ARG 4 rateLimiter
```
An leftover name. Originally it appears like tracking period because it leaves timestamps and returns time differences